src/components/map/mapbox/controls/menu/MenuControl.tsx

```diff
@@ -23,7 +23,7 @@ interface MenuControlProps {
 class MapControl implements IControl {
   private container: HTMLDivElement | null = null;
   private root: Root | null = null;
-  private readonly component: MapControlType;
+  private component: MapControlType;
```

Comment:
This must be readonly, if you check previous commit, Lyn make similar changes and break other area, please rollback changes of this file